### PR TITLE
Don't limit to 20 groups

### PR DESCRIPTION
**Pull request by mestachs**
## Self proof reading checklist

- [ ] Did I run Pronto and fixed all warnings (pronto run -c origin/dev)
- [ ] Make sure all naming and code will remain understandable in 6 month by someone new to the project or by you.
- [ ] Did I test the right thing?
- [ ] Did I test corner cases, non happy path (defensive testing)?
- [ ] Check that I used the ad-hoc patterns and created files accordingly (Presenters, Services, Search object, Form object,...)
- [ ] Check code efficiency with record intensive project
- [ ] Update documentation,readme accordingly

Thanks!

### CODE REVIEW

File: app/controllers/setup/autocomplete_controller.rb [hunk 1]
```diff
@@ -5,7 +5,7 @@ def organisation_unit_group
     if params.key?(:term)
       organisation_unit_group_by_term_on_sol
     elsif params.key?(:siblings)
-      results = search_results(nil, "organisation_unit_groups")
+      results = search_results(nil, "organisation_unit_groups", limit: 20_000)
```
Comment (pjaspers) [inline app/controllers/setup/autocomplete_controller.rb]: Now that's a big limit.

Comment (mestachs) [reply app/controllers/setup/autocomplete_controller.rb]: just aligned to the one for data elements
